bundles/edu.kit.ipd.sdq.pcm.simulation.scheduler.exact/src/edu/kit/ipd/sdq/pcm/simulation/scheduler/exact/factory/Windows7SchedulerFactory.java

```diff
@@ -3,14 +3,16 @@
 import de.uka.ipd.sdq.scheduler.IActiveResource;
 import de.uka.ipd.sdq.scheduler.SchedulerModel;
 import de.uka.ipd.sdq.scheduler.factory.SchedulerExtensionFactory;
+import de.uka.ipd.sdq.scheduler.resources.active.IResourceTableManager;
 import edu.kit.ipd.sdq.pcm.simulation.scheduler.exact.ExactSchedulingFactory;
 
 
 public class Windows7SchedulerFactory implements SchedulerExtensionFactory {
 
     @Override
-    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores) {
+    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores
+            , IResourceTableManager resourceTableManager) {

```

Comment:
Same, is there maybe some bug in the Palladio formatter rules? https://github.com/PalladioSimulator/Palladio-Build-CodingConventions/tree/master/misc/org.palladiosimulator.codeconventions

Reply:
worked in review finding; reformatted file using Palladio code formatter